soccer/gameplay/robocup-py.cpp

```diff
@@ -265,6 +436,7 @@ BOOST_PYTHON_MODULE(robocup)
 	boost::python::register_exception_translator<NullArgumentException>(&translateException);
 
 	def("fix_angle_radians", &fixAngleRadians);
+	def("get_trapezoidal_time", &Trapezoidal::getTime);
```

Comment:
Yeah I guess so.  I was just thinking something along the lines of "estimate_travel_time" - something more related to it's purpose in the program than it's implementation.